rdma_srv/src/rdma_agent.rs

```diff
@@ -626,6 +626,10 @@ impl RDMAAgent {
                 let vpcId;
                 let ipAddr;
                 if RDMA_CTLINFO.isK8s {
+                    while self.GetVpcId() == 0 {
+                        println!("Waiting for control plane to get vpcId");
+                        sleep(Duration::from_millis(1000));
```

Comment:
+TODO: need find a workaround instead of loop here.

Reply:
Added